doc/guides/kconfig/index.rst

```diff
@@ -464,6 +464,18 @@ Here are some things to check:
   they come from.
 
 
+Checking changes with :file:`scripts/kconfig/lint.py`
+*****************************************************
+
+The script :zephyr_file:`scripts/kconfig/lint.py` can be run directly to run
```

Comment:
How about:
```
After you make Kconfig changes, you can use the
:zephyr_file:`scripts/kconfig/lint.py` script to perform
useful Kconfig checks and potential issues:  use 
``--help`` to see available check options.
```

Reply:
Rephrased it to something similar and added some more information.